tests/tck/features/path/NoLoop.feature

```diff
@@ -242,3 +242,40 @@ Feature: NoLoop Path
       | [[:like "Tony Parker"->"Tim Duncan" @0 {}]]                                                         |
       | [[:like "LaMarcus Aldridge"->"Tim Duncan" @0 {}], [:like "Tony Parker"->"LaMarcus Aldridge" @0 {}]] |
       | [[:like "Manu Ginobili"->"Tim Duncan" @0 {}], [:like "Tony Parker"->"Manu Ginobili" @0 {}]]         |
+
+  Scenario: Query with NO LOOP on a node without self-loop
+    When executing query:
+      """
+      CREATE SPACE TestNoLoopSpace(vid_type=fixed_string(20));
+      USE TestNoLoopSpace;
+      """
+    And wait 3 seconds
+    Then the execution should be successful
+    When executing query:
+      """
+      CREATE TAG Person(name string);
+      CREATE EDGE Link();
+      INSERT VERTEX Person(name) VALUES "nodea":("Node A");
+      INSERT EDGE Link() VALUES "nodea" -> "nodea":();
+      """
+    And wait 1 seconds
+    Then the execution should be successful
+    When executing query:
+      """
+      FIND NOLOOP PATH FROM "nodea" TO "nodea" OVER Link YIELD PATH AS p;
+      """
+    Then the result should be, in any order:
+      | p     |
+      | EMPTY |
```

Comment:
The result seems an empty table rather than a table contains EMPTY value, such as: 

```feature
    Then the result should be, in any order:
      | p     |
```